### DISCUSSION TITLE
A Hacker changed my email and password, I opened a ticket two weeks ago, but no response

### DISCUSSION THREAD

**Discussion by Shuijing123** [community/community#108646] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Two weeks ago, my other account was hacked. The hacker changed my primary email AND password within 2 minutes.
 I cannot login or recover this account since then. My account has many public repos that other users are actively using, but I can't maintain them since then.

After searching online, I opened a ticket under "Github support". But no response after 2 weeks. 

It's really disappointing to see the unreasonably long wait time for github tickets from [this post](https://github.com/orgs/community/discussions/23989). How long does it take to resolve issues similar to mine?

My ticket # is 2571861.

**Comment by asmawi117** [comment]
same issue, 35 days without answer from github support

**Comment by Shuijing123** [comment]
Update: 29 days later, github support resolved my issue. I guess there's not too much we can do except waiting :(

**Comment by 8b1tz** [comment]
:(

**Comment by Snowtuss** [comment]
When we send a ticket we get "We are experiencing high volumes and therefore, you may experience longer than normal wait times" so i am sure they will get to it, but it will take some time sadly :(. 
If you don't mind me asking, do you know how did you get hacked ?
